ownCloud/Server List/ServerListTableViewController.swift

```diff
@@ -102,6 +102,56 @@ class ServerListTableViewController: UITableViewController, Themeable {
 		}
 
 		ReleaseNotesDatasource.setUserPreferenceValue(NSString(utf8String: VendorServices.shared.appVersion), forClassSettingsKey: .lastSeenAppVersion)
+
+		messageCountSelector = MessageSelector(filter: nil, handler: { [weak self] (messages, _, _) in
+			var countByBookmarkUUID : [UUID? : Int ] = [:]
+
+			if let messages = messages {
+				for message in messages {
+					if !message.resolved {
+						if let count = countByBookmarkUUID[message.bookmarkUUID] {
+							countByBookmarkUUID[message.bookmarkUUID] = count + 1
+						} else {
+							countByBookmarkUUID[message.bookmarkUUID] = 1
+						}
+					}
+				}
+			}
+
+			OnMainThread {
+				self?.messageCountByBookmarkUUID = countByBookmarkUUID
+			}
+		})
+	}
+
+	var messageCountSelector : MessageSelector?
+
+	static let BookmarkMessageCountChanged = NSNotification.Name("boomark.message-count.changed")
```

Comment:
Should you add a constant for this value: `"boomark.message-count.changed"`
Please check all other files, where this value was used.

Reply:
Moved this to its own `NSNotification.Name` extension now. Thanks.